core/cli/src/lib.rs

```diff
@@ -47,14 +47,15 @@ pub struct CheckingVersionError {
 pub type Result<T, E = Error> = std::result::Result<T, E>;
 
 pub struct AxonCli {
-    version: Version,
-    matches: ArgMatches,
+    kernel_version:      Version,
+    application_version: Version,
+    matches:             ArgMatches,
 }
 
 impl AxonCli {
-    pub fn init(axon_version: Version, cli_version: &'static str) -> Self {
+    pub fn init(kernel_version: Version, application_version: Version) -> Self {
         let matches = Command::new("axon")
-            .version(cli_version)
+            .version(kernel_version.to_string())
```

Comment:
I'm not sure this should be changed to kernel_version. IMO it should be left to the library user. They can still include axon version in app version as build info, e.g. `my-chain 0.2.0+aabbccdd.axon.0.1.0`.

Reply:
This also means the cli won't have git info.